build.assets/build-package.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-usage() { echo "Usage: $(basename $0) [-t <oss/ent>] [-v <version>] [-p <package type>] <-a [amd64/x86_64]|[386/i386]> <-r go1.9.7|fips> <-s tarball source dir> <-m tsh>" 1>&2; exit 1; }
+usage() { echo "Usage: $(basename "$0") [-t <oss/ent>] [-v <version>] [-p <package type>] <-a [amd64/x86_64]|[386/i386]> <-r go1.9.7|fips> <-s tarball source dir> <-m tsh>" 1>&2; exit 1; }
```

Comment:
```suggestion
# shellcheck disable=SC2086
usage() { echo "Usage: $(basename $0) [-t <oss/ent>] [-v <version>] [-p <package type>] <-a [amd64/x86_64]|[386/i386]> <-r go1.9.7|fips> <-s tarball source dir> <-m tsh>" 1>&2; exit 1; }
```
I had to do a similar thing to this in the node join script - escaping the quotes (`\"`) is untidy and doesn't actually work as intended, plus you still need the double quotes around the whole thing. Adding multiple quotes just expands the variables outside of the quotes and does the opposite of what the linter actually intends.